packages/zoe/src/contractSupport/zoeHelpers.js

```diff
@@ -195,6 +195,39 @@ export const trade = (
   }
 };
 
+const findFromOtherSeat = (
```

Comment:
I'd like your help to try to answer the following question:

Would this same trick, introducing `findFromOtherSeat`, solve the problem that was blocking #1905 ?

(An answer of "yes" does not mean we should do #1905 . It remains too complicated. But it would help me understand how these PRs relate and whether there's a way to get the best of both.)

Reply:
See https://github.com/Agoric/agoric-sdk/pull/1905#issuecomment-716842929